diesel_compile_tests/tests/compile-fail/combinations_require_same_sql_types.rs

```diff
@@ -0,0 +1,39 @@
+#[macro_use]
```

Comment:
I would prefer to have this as `ui` test, as we plan to migrate all our compile tests to `ui`/`trybuild` at some point in time